react/features/contact-list/components/ContactListPanel.web.js

```diff
@@ -117,49 +70,14 @@ class ContactListPanel extends Component {
 
             return (
                 <ContactListItem
-                    avatarURI = { interfaceConfig.SHOW_CONTACTLIST_AVATARS
+                    avatarURI = { this.props._showAvatars
                         ? getAvatarURL(participant) : null }
                     id = { id }
                     key = { id }
                     name = { name } />
             );
         });
     }
-
-    /**
-     * Renders a React Element for informing the conference is currently locked.
-     *
-     * @private
-     * @returns {ReactElement}
-     */
-    _renderLockedMessage() {
-        return (
-            <p
-                className = 'form-control__hint form-control_full-width'
-                id = 'contactListroomLocked'>
-                <span className = 'icon-security-locked' />
-                <span>{ this.props.t('roomLocked') }</span>
```

Comment:
"roomLocked" string doesn't look to be used anymore and can also be removed from the translation file.